src/site/layouts/campaign_landing_page.drupal.liquid

```diff
@@ -209,7 +209,7 @@
                   <h3 class="vads-u-margin-top--0">
                     <a
                       href="{{ linkTeaser.entity.fieldLink.uri }}"
-                      onclick="recordEvent({ 'event': 'clp-link-click', 'clp-section-category': 'Spotlight', 'clp-section-title': '{{ fieldClpSpotlightHeader | escape }}', 'clp-total-sections': '{{ clpTotalSections }}', 'clp-click-label': '{{ linkTeaser.entity.fieldLink.title | escape }}'"
+                      onclick="recordEvent({ 'event': 'clp-link-click', 'clp-section-category': 'Spotlight', 'clp-section-title': '{{ fieldClpSpotlightHeader | encode }}', 'clp-total-sections': '{{ clpTotalSections }}', 'clp-click-label': '{{ linkTeaser.entity.fieldLink.title | encode }}' });"
```

Comment:
This was the big gotcha, a missing `});` here

Reply:
Was this the culprit behind the runtime error? TBH I wonder if we can just do this change instead of introducing a new filter for encoding. I _think_ the CMS already does the HTML entity-encoding but really it made my heart jump thinking that we may have been escaping data wrong in Liquid templates all of this time

Reply:
It wasn't the only culprit I think, in Brian's screenshot there were definitely single quotes that were being read as end of input for the onclick property :/